qiskit/transpiler/passes/routing/cython/stochastic_swap/utils.pyx

```diff
@@ -162,32 +162,27 @@ cdef class NLayout:
         return out
     
     
-cpdef NLayout nlayout_from_layout(object layout, object qregs, 
+cpdef NLayout nlayout_from_layout(object layout,
+                                  object qubit_indices,
```

Comment:
I would make this a dict (assuming the signature declaration in the pxd is updated to match:

```suggestion
                                  dict qubit_indices,
```

I don't think it matters too much, but feels like explicitly typing the argument to always be a dict makes the most sense instead of taking any object. (I expect it was object before because an `OrderedDict` was used for python 3.5 compat)

Reply:
Agree, added in 23a6597 .